tools/manage-osqueryd.ps1

```diff
@@ -95,7 +101,22 @@ function Do-Service {
     } else {
       Write-Host "'$kServiceName' is not an installed system service." -foregroundcolor Yellow
       Exit 1      
-    }  
+    }
+  } elseif ($install_wel_manifest) {
+    wevtutil im $windowsEvengLogManifest
```

Comment:
Maybe here you could do a
```
if (-not (Test-Path $windowsEvengLogManifest)) {
  Write-Host "[-] Failed to find osquery Event Log manifest file!" -ForegroundColor Red
  ...
}
```
or the like? Something to indicate that it couldn't obtain the manifest location _before_ attempting to install it?